addons/l10n_it_edi_withholding/tests/test.xml

```diff
@@ -0,0 +1,142 @@
+      </DatiGeneraliDocumento>
+    </DatiGenerali>
+    <DatiBeniServizi>
+      <DettaglioLinee>
+        <NumeroLinea>1</NumeroLinea>
+        <Descrizione>
+            Ordinary accounting service for the year
+        </Descrizione>
+        <Quantita>1.00</Quantita>
+        <PrezzoUnitario>350.000000</PrezzoUnitario>
+        <PrezzoTotale>350.00</PrezzoTotale>
+        <Ritenuta>SI</Ritenuta>
+        <AliquotaIVA>22.00</AliquotaIVA>
+        </DettaglioLinee>
+      <DettaglioLinee>
+        <NumeroLinea>2</NumeroLinea>
+        <Descrizione>
+            Balance deposit for the past year
+        </Descrizione>
+        <Quantita>1.00</Quantita>
+        <PrezzoUnitario>300.000000</PrezzoUnitario>
+        <PrezzoTotale>300.00</PrezzoTotale>
+        <Ritenuta>SI</Ritenuta>
+        <AliquotaIVA>22.00</AliquotaIVA>
+        </DettaglioLinee>
+      <DettaglioLinee>
+        <NumeroLinea>3</NumeroLinea>
+        <Descrizione>
+            Ordinary accounting service for the trimester
+        </Descrizione>
+        <Quantita>1.00</Quantita>
+        <PrezzoUnitario>50.000000</PrezzoUnitario>
+        <PrezzoTotale>50.00</PrezzoTotale>
+        <Ritenuta>SI</Ritenuta>
+        <AliquotaIVA>22.00</AliquotaIVA>
+        </DettaglioLinee>
+      <DettaglioLinee>
+        <NumeroLinea>4</NumeroLinea>
+        <Descrizione>
+            Electronic invoices management
+        </Descrizione>
+        <Quantita>1.00</Quantita>
+        <PrezzoUnitario>50.000000</PrezzoUnitario>
+        <PrezzoTotale>50.00</PrezzoTotale>
+        <Ritenuta>SI</Ritenuta>
+        <AliquotaIVA>22.00</AliquotaIVA>
+        </DettaglioLinee>
+                <DettaglioLinee>
+                    <NumeroLinea>5</NumeroLinea>
+                    <Descrizione>Contributo ENASARCO</Descrizione>
+                    <PrezzoUnitario>0.00</PrezzoUnitario>
+                    <PrezzoTotale>0.00</PrezzoTotale>
+                    <AliquotaIVA>22.00</AliquotaIVA>
+                    <AltriDatiGestionali>
+                        <TipoDato>CASSA-PREV</TipoDato>
+                        <RiferimentoTesto>TC07 - ENASARCO</RiferimentoTesto>
+                        <RiferimentoNumero>63.75</RiferimentoNumero>
+                    </AltriDatiGestionali>
+                </DettaglioLinee>
+      <DatiRiepilogo>
+        <AliquotaIVA>22.00</AliquotaIVA>
+        <ImponibileImporto>750.00</ImponibileImporto>
+        <Imposta>165.00</Imposta>
+        <EsigibilitaIVA>I</EsigibilitaIVA>
+      </DatiRiepilogo>
+    </DatiBeniServizi>
+    <DatiPagamento>
+      <CondizioniPagamento>TP02</CondizioniPagamento>
+      <DettaglioPagamento>
+        <ModalitaPagamento>MP05</ModalitaPagamento>
+        <DataScadenzaPagamento>2022-03-24</DataScadenzaPagamento>
+        <ImportoPagamento>765.00</ImportoPagamento>
+        <CodicePagamento>INV/2022/03/0010</CodicePagamento>
+      </DettaglioPagamento>
+    </DatiPagamento>
+  </FatturaElettronicaBody>
+</p:FatturaElettronica>
```

Comment:
Can you add a newline at EOF? Even on XMLs please.

Reply:
It was a test file that I forgot to remove :sweat_smile: